packages/zelus/zelus.2.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "zelus"
+version: "2.1"
+synopsis: "A synchronous language with ODEs"
+authors: ["Marc Pouzet <marc.pouzet@ens.fr>"]
+maintainer: ["Marc Pouzet <marc.pouzet@ens.fr>"]
+homepage: "http://zelus.di.ens.fr"
+doc: "http://zelus.di.ens.fr/man/"
+bug-reports: "https://gitlab.inria.fr/parkas/zelus/-/issues"
+depends: [
+  "ocaml" {>= "4.08.1"}
+  "dune" {>= "2.6" & build}
```

Comment:
dune should not be tagged as a build dependency (see https://github.com/ocaml/dune/issues/2147)